Samples/Samples/View/CalendarEventPage.xaml

```diff
@@ -6,63 +6,61 @@
              xmlns:essentials="clr-namespace:Xamarin.Essentials;assembly=Xamarin.Essentials"
              xmlns:converters="clr-namespace:Samples.Converters;assembly=Samples"
              x:Class="Samples.View.CalendarEventPage"
-             x:DataType="essentials:IEvent"
+             x:DataType="essentials:Event"
```

Comment:
Does DataType now work with IEvent?

Reply:
The reason I've changed it to Event instead of IEvent, is that there were variables added to Event that are not required for an IEvent. 

e.g:
public DateTimeOffset? StartDate => Start.HasValue ? (DateTimeOffset?)DateTimeOffset.FromUnixTimeMilliseconds((long)Start.Value).ToLocalTime() : null;

This allows me to view a datetimeoffset version of the Start epoch for date listing purposes, but it's not a requirement, i.e. the user who uses the IEvent interface is not required to have datetimeoffset version of this value.